x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -0,0 +1,233 @@
+              and: [],
+              enabled: true,
+              id,
+              name: domain,
+              excluded: false,
+              kqlQuery: '',
+              queryMatch: {
+                field: domainAttr,
+                value: domain,
+              },
+              queryDate: {
+                from: startDate,
+                to: Date.now(),
+              },
+            }}
+            render={(dataProvider, _, snapshot) =>
+              snapshot.isDragging ? (
+                <DragEffects>
+                  <Provider dataProvider={dataProvider} />
+                </DragEffects>
+              ) : (
+                <>
+                  {domain}
+                  {domains.length > 1 && (
+                    <EuiToolTip
+                      content={
+                        <>
+                          {domains.slice(1, 6).map(domainName => (
+                            <span key={`${id}-${domainName}`}>
+                              {defaultToEmptyTag(domainName)}
+                              <br />
+                            </span>
+                          ))}
+                          {domains.slice(1).length > 5 && (
+                            <b>
+                              <br />
+                              more ...
+                            </b>
+                          )}
+                        </>
+                      }
+                    >
+                      <MoreDomains type="eye" />
+                    </EuiToolTip>
+                  )}
+                </>
+              )
+            }
+          />
+        );
+      } else {
+        return getEmptyTagValue();
+      }
+    },
+  },
+  {
+    name: i18n.DIRECTION,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) =>
+      has('network.direction', node)
+        ? get('network.direction', node).length > 0
+          ? get('network.direction', node).map((direction: NetworkDirectionEcs) => (
+              <AddToKql
+                key={escapeDataProviderId(
+                  `networkTopNFlow-table-${topNFlowType}-${topNFlowDirection}-direction-${direction}`
+                )}
+                content={i18n.FILTER_TO_KQL}
+                expression={`network.direction: ${escapeQueryValue(direction)}`}
+                type={type}
+              >
+                {defaultToEmptyTag(direction)}
+              </AddToKql>
+            ))
+          : getEmptyTagValue()
+        : getEmptyTagValue(),
+  },
+  {
+    name: i18n.BYTES,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) =>
+      has('network.bytes', node)
+        ? numeral(node.network!.bytes).format('0.000b')
+        : getEmptyTagValue(),
+  },
+  {
+    name: i18n.PACKETS,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) =>
+      has('network.packets', node)
+        ? numeral(node.network!.packets).format('0,000')
+        : getEmptyTagValue(),
```

Comment:
As a side note (I haven't gotten to the query yet), but I would imagine this would be returning packets for `source ip` and `client ip`. That might be accurate but looks suspicious to me.

Reply:
no!!! it is not!!!